sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdClientChannelPool.java

```diff
@@ -1116,19 +1113,20 @@ private void newTimeout(
         this.acquisitionAndIdleEndpointDetectionTimeout.set(acquisitionAndIdleEndpointDetectionTimer.newTimeout(
             (Timeout timeout) -> {
                 if (idleEndpointTimeoutInNanos == 0) {
-                    if (logger.isDebugEnabled()) {
-                        logger.debug("Idle endpoint check is disabled");
+                    // log is too noisy when debug is enabled
+                    if (logger.isTraceEnabled()) {
+                        logger.trace("Idle endpoint check is disabled");
                     }
                 } else {
                     final long elapsedTimeInNanos = System.nanoTime() - endpoint.lastRequestNanoTime();
 
                     if (idleEndpointTimeoutInNanos - elapsedTimeInNanos <= 0) {
-                        if (logger.isDebugEnabled()) {
+                        if(logger.isDebugEnabled()) {
```

Comment:
nit: add back the space after if as it was before for consistency and readablity